rts/Lua/LuaConstGL.cpp

```diff
@@ -392,19 +412,22 @@ bool LuaConstGL::PushEntries(lua_State* L)
 	/*** @field GL.QUADRATIC_ATTENUATION integer */
 	PUSH_GL(QUADRATIC_ATTENUATION);
 
-	/***
-	 * Shader Types
-	 * @section shadertypes
-	 */
+	/*** @alias GL.ShaderType
+	  * | GL.VERTEX_SHADER
+	  * | GL.TESS_CONTROL_SHADER
+	  * | GL.TESS_EVALUATION_SHADER
+	  * | GL.GEOMETRY_SHADER
+	  * | GL.FRAGMENT_SHADER
+	  */
 
 	/*** @field GL.VERTEX_SHADER integer */
 	PUSH_GL(VERTEX_SHADER);
 	/*** @field GL.TESS_CONTROL_SHADER integer */
 	PUSH_GL(TESS_CONTROL_SHADER);
 	/*** @field GL.TESS_EVALUATION_SHADER integer */
 	PUSH_GL(TESS_EVALUATION_SHADER);
-	/*** @field GL.GEOMETRY_SHADER_EXT integer */
-	PUSH_GL(GEOMETRY_SHADER_EXT);
+	/*** @field GL.GEOMETRY_SHADER integer */
```

Comment:
Based on [old changelog](https://github.com/beyond-all-reason/RecoilEngine/blob/c3fe612/doc/changelog.txt#L791) I legitimately believe this was a mistake. See also the alias above used in the mentioned functions.

Reply:
I dug all recoil game repos I could find, none has references to this constant. I believe we don't require exposing both.